runners/core-java/src/main/java/org/apache/beam/runners/core/metrics/MetricsContainerImpl.java

```diff
@@ -108,6 +126,11 @@ private void reset(MetricsMap<?, ? extends MetricCell<?>> cells) {
     }
   }
 
+  @Override
+  public boolean isProcessWide() {
```

Comment:
Is the only use of this to prevent it from being reset? (Which, presumably, would be easy to accidentally do?) If so, can we make this private (or just access the member directly) and remove the addition to the interface?

Reply:
Done. Made it private

(Yes. Previously there was a PR which had called reset() on it which caused some debugging issued when adding this cl.)